smos-docs-site/pages/concepts.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: SMOS Concepts
+date: 2020-04-08
+---
+
+GTD
+: GTD is a time management method by David Allen (see [Getting This
+  Done](https://en.wikipedia.org/wiki/Getting_Things_Done) on
+  Wikipedia).
+  SMOS is supposed to be a tool to implement GTD.
+
+
+<!-- the next concepts are related to SMOS configuration, but AFAIK
+are not explained -->
+
+Workflow
+: What's the meaning in SMOS and why the need for a directory?
+
+
+Projects
+: Whats the meaning in SMOS?  Why the need for a directory in the
+  configuration file?
+
+
+Archive:
+: When task and projects are completed, they are archived (i.e. moved
+  in the archive directory).
+
+State File
+: ???
+
+Entry
+: In SMOS each entry can be in one of the following states: <!-- not true. an entry can have no state -->
+
+	- WAITING
+	- TODO
+	- STARTED
+	- READY
+	- NEXT
+	- FAILED
+	- DONE
+	- CANCELLED
+
+
+Forest
+: In SMOS each forest can be in one of the following states: <!-- not true, a forest can have no state  -->
```

Comment:
No, states only exist for entries, not forests.
A smos file is defined as a forest of entries.
Literally:

```
newtype SmosFile = SmosFile { smosFileForest :: Forest Entry }
```